lib/omnibus/software.rb

```diff
@@ -571,11 +571,17 @@ def with_standard_compiler_flags(env = {}, opts = {})
           freebsd_flags
         when "windows"
           arch_flag = windows_arch_i386? ? "-m32" : "-m64"
-          opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
+          #opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
```

Comment:
I think having your comments is fine, but you should just delete the commented out code. You can always come back and get it